### PR TITLE
initial draft of script to trigger multiple specific granules

### PR DESCRIPTION
**Pull request by niarenaw**
## Purpose
- script to trigger DSWx production from list of granukes
## Issues
- [issue 15](https://github.com/nasa/opera-sds-ops/issues/15)
## Testing
- Tested extensively with numerous PST processing requests

### CODE REVIEW

**Comment by philipjyoon** [comment]
Looks great. Just a few minor comments:

1. Would be good to print out in help text what a granule ID looks like so people less familiar will have confidence. Also granule IDs are space-separated list? Would be good to have a command example w a couple real granule IDs.
2. Number of granules would be limited by the command line max on the Mozart box. Would be good to find out what that is and state that in the help text.
3. I don't know shell programming too well but it would be better if the command string is constructed once and it's used for both dryrun and normal modes. That way you don't have two different lines of code that's constructing the same command. Prevents them from ever being out of sync in future.

File: trigger_dswx_from_list_of_granules.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		collection="HLSS30"
+	elif [[ ${granule:4:1} == "L" ]]; then
+		collection="HLSL30"
+	else
+		echo "invalid granule format: $granule"
+		exit 1
+	fi
+
+	if $DRYRUN; then
+		echo "~/mozart/ops/opera-pcm/data_subscriber/daac_data_subscriber.py query -c $collection -p LPCLOUD --release-version=$RELEASE_VERSION --job-queue=opera-job_worker-hls_data_download --chunk-size=1 --native-id=$granule"
```
Comment (riverma) [inline trigger_dswx_from_list_of_granules.sh:31]: If the `daac_data_subscriber.py` script moves, this script will break. Do we have environment variables we can use to point to the `opera-pcm` directory?

File: trigger_dswx_from_list_of_granules.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	else
+		~/mozart/ops/opera-pcm/data_subscriber/daac_data_subscriber.py query -c $collection -p LPCLOUD --release-version=$RELEASE_VERSION --job-queue=opera-job_worker-hls_data_download --chunk-size=1 --native-id=$granule
```
Comment (riverma) [inline trigger_dswx_from_list_of_granules.sh:33]: Similar to comment above.

**Review by riverma — COMMENTED** [review]
@niarenaw this is really nice. Agree with @philipjyoon's comment about need for an example of how to run this script, that would clear up the comma vs space delimiter question. Nothing blocking this from merge, but some suggestions below.

File: trigger_dswx_from_list_of_granules.sh (hunk 1 above)
Comment (riverma) [reply trigger_dswx_from_list_of_granules.sh:31]: CC @hhlee445 on above question about environment variables.

Comment (niarenaw) [reply trigger_dswx_from_list_of_granules.sh:41]: Good point -- added some basic help text with an example.

**Comment by riverma** [comment]
LGTM! Thanks @niarenaw.

**Pull request merged**